src/styles/themes/default.js

```diff
@@ -1,52 +1,54 @@
 import colors from '../colors';
 
 export default {
-    shadow: colors.black,
-    link: colors.blue,
-    linkHover: colors.blueHover,
-    componentBG: colors.white,
-    hoverComponentBG: colors.gray1,
-    activeComponentBG: colors.gray2,
-    appBG: colors.white,
-    heading: colors.dark,
-    sidebar: colors.gray1,
-    sidebarHover: colors.white,
-    border: colors.gray2,
+    // Figma keys
+    appBG: colors.greenAppBackground,
+    highlightBG: colors.greenHighlightBackground,
+    border: colors.greenBorders,
     borderFocus: colors.blue,
-    icon: colors.gray3,
-    iconHovered: colors.dark,
+    icon: colors.greenIcons,
+    iconHovered: colors.white,
     iconSuccessFill: colors.green,
-    iconReversed: colors.white,
-    textSupporting: colors.gray4,
-    text: colors.dark,
+    iconReversed: colors.greenAppBackground,
+    textSupporting: colors.greenSupportingText,
+    text: colors.white,
+    link: colors.blue,
+    linkHover: colors.blueHover,
+    buttonDefaultBG: colors.greenDefaultButton,
+    buttonDisabledBG: colors.greenDefaultButtonDisabled,
+    buttonHoveredBG: colors.greenDefaultButtonHover,
+    buttonPressedBG: colors.greenDefaultButtonPressed,
+    danger: colors.red,
+    dangerHover: colors.redHover,
+    dangerPressed: colors.redHover,
+    dangerDisabled: colors.redDisabled,
+    success: colors.green,
+    successHover: colors.greenHover,
+    successPressed: colors.greenPressed,
+    transparent: colors.transparent,
+
+    // Additional keys
+    inverse: colors.white,
+    shadow: colors.black,
+    componentBG: colors.greenAppBackground,
+    hoverComponentBG: colors.greenHighlightBackground,
+    activeComponentBG: colors.greenBorders,
+    sidebar: colors.greenHighlightBackground,
+    sidebarHover: colors.greenAppBackground,
+    heading: colors.white,
+    textLight: colors.white,
+    textDark: colors.greenAppBackground,
+    textReversed: colors.greenAppBackground,
+    textBackground: colors.greenHighlightBackground,
+    textMutedReversed: colors.greenIcons,
     textError: colors.red,
-    textSuccess: colors.green,
-    textBackground: colors.gray1,
-    textReversed: colors.white,
-    textMutedReversed: colors.gray3,
-    buttonDefaultBG: colors.gray2,
-    buttonSuccessBG: colors.green,
-    buttonSuccessHoveredBG: colors.greenHover,
-    buttonDangerBG: colors.red,
-    buttonDangerPressedBG: colors.redHover,
-    buttonDangerDisabledBG: colors.redDisabled,
-    online: colors.green,
-    offline: colors.gray3,
-    sidebarButtonBG: 'rgba(198, 201, 202, 0.25)',
-    opaqueAvatar: 'rgba(011, 027, 052, 0.64)',
-    modalBackdrop: colors.gray3,
-    modalBackground: colors.gray2,
-    badgeDangerBG: colors.red,
-    badgeDangerPressedBG: colors.redHover,
-    badgeDefaultBG: colors.gray2,
-    badgeSuccessBG: colors.green,
```

Comment:
There were about 3 sets of theme keys for the same success green/danger red colors. I didn't remove all of them (i.e textError) but I decided that was the best place to cut this file down a bit.